### DISCUSSION TITLE
Linguist-language syntax highlight preview in pull requests unstable (on refresh sometimes plain text)

### DISCUSSION THREAD

**Discussion by ZoranMedojevic** [github-linguist/linguist#6817] [Classification]
Hi guys, first time commenting, very glad i found this project. great staff :)
Started using `linguist-language` yesterday, almost there but maybe I'm missing something :sweat_smile: 

### Context
I want to have `.isml` files highlighted as HTML in my GitHub pull requests.
That would help a lot during code reviews.

### I used
![image](https://github.com/github-linguist/linguist/assets/11091071/291181e5-8dd5-466e-bc6f-1b7520b1a6ff)

### How to reproduce
Visit this [pull request](https://github.com/ZoranMedojevic/test/pull/9/files), and [demo project](https://github.com/ZoranMedojevic/test). 

### Issue

Sometimes pull requests show correct syntax for `.isml` files `but after refresh it shows plain text`.

<details><summary>Img - correct preview (CLICK ME)</summary>
<br>

<img src="https://github.com/github-linguist/linguist/assets/11091071/27649da4-3c9d-445d-8762-4efb4b901387" width="100%">

</details>

<details><summary>Img - wrong preview (plain text) (CLICK ME)</summary>
<br>

<img src="https://github.com/github-linguist/linguist/assets/11091071/869193b1-4bcf-4844-93c5-55d11d3e0210" width="100%">

</details>

<details><summary>Video demo (CLICK ME)</summary>
<br>

https://github.com/github-linguist/linguist/assets/11091071/2493c07d-ac4c-44d3-9770-c9067295b059

</details>

**Comment by lildude** [comment]
This looks like a rendering issue somewhere outside of Linguist's control. The HTML code is correctly showing the language has been identified as HTML indicating Linguist has done its job, so something in the rendering is getting stuck.

Please use the "Conect" link at the bottom of the page to report this to GitHub support and they'll route the issue to the correct team.

**Reply by ZoranMedojevic** [reply]
@lildude thanks for quick response.
Deal, will do that.
